scripts/Imaging/imgpy/docs/filters.md

```diff
@@ -0,0 +1,135 @@
+# Available pre-processing of input raw images/projections:
+## Region of interest
+```
+  -R [REGION_OF_INTEREST [REGION_OF_INTEREST ...]], --region-of-interest [REGION_OF_INTEREST [REGION_OF_INTEREST ...]]
+                        Crop original images using these coordinates, after rotating the images.
+                        If not given, the whole images are used.
+                        Example: --region-of-interest='[150,234,23,22]'.
```

Comment:
The order is `[Left, Top, Right, Bottom]`, I've added it to the docs. This is the standard format used in the scripts, and the current and future GUI and in Octopus.